packages/mdc-menu-surface/util.ts

```diff
@@ -21,16 +21,12 @@
  * THE SOFTWARE.
  */
 
-/** @type {string|undefined} */
-let storedTransformPropertyName_;
+let storedTransformPropertyName_: string | undefined;
 
 /**
  * Returns the name of the correct transform property to use on the current browser.
- * @param {!Window} globalObj
- * @param {boolean=} forceRefresh
- * @return {string}
  */
-function getTransformPropertyName(globalObj, forceRefresh = false) {
+function getTransformPropertyName(globalObj: Window, forceRefresh = false): string {
```

Comment:
The return type should probably just be `transform | webkitTransform`, right? What are the other options?

Reply:
Done